events/omero-users-workshop-turku-online-november-2021.html

```diff
@@ -0,0 +1,44 @@
+---
+layout: news-events
+filename: events
+title: November 2021 OMERO users workshop
+main-blurb: 2-day workshop on how to use OMERO held remotely, organized by Åbo Akademi, Turku.
+---          
+     <hr class="whitespace">
+        <div class="row">
+            <div class="column">
+                <h2 class="event-day">Organiser: <a href="https://www.abo.fi/" target="_blank">Åbo Akademi</a></h2>
+            </div>
+        </div>
+        <div class="row">
+            <div class="column">
+                <h2 class="event-day">Location: held remotely
+            </div>
+        </div>
+        <div class="row">
+            <div class="column">
+                <h2 class="event-day">Date: 15-16th November 2021</h2>
+            </div>
+        </div>
+        <hr class="whitespace">
+        <div class="row">
+            <div class="column">
+                <h2 class="event-day">Duration: 3 hours per day</h2>
+                <p class="event-day-description">The Open Microscopy Environment (OME) is an open-source software project that develops tools that enable access, analysis, visualization, sharing and publication of biological image data. We support more than 150 image data formats across many imaging modalities including medical imaging, high-content screening and whole-slide imaging.
+                We support and visualize multi-channel 3D timelapse images.
+ 
+                OMERO, a component of OME, is a software platform for image data management and analysis which was presented in this course.
+                Import, organisation, viewing, searching, annotation, analysis with Fiji and publishing of imaging data with OMERO was shown.
+                </p>
+                <p class="event-day-description">
+                The workshop concentrated on basics of OMERO and did not assume any programming skills.
```

Comment:
This is in the past-tense but I assume since it's within a week that's fine.

Reply:
If we wait till tomorrow, it will become a past tense :)